### PR TITLE
Fixes for Ruby 3

### PR DESCRIPTION
**Pull request by grantspeelman**
I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by arunagw** [comment]
@grantspeelman thanks for the fixes for Ruby 3. We would love to get these changes merged but it seems there are other things failing currently on Ruby 3. Can be seen in #286.

File: gem_config.rb [hunk 1]
```diff
@@ -36,6 +36,7 @@ def self.metadata
 
   def self.dev_dependencies
     [["bundler", ">= 1.3.0"],
+     ["mime-types", ">= 3.0.0"],
```
Comment (grantspeelman) [inline gem_config.rb:39]: @arunagw made this only a dev dependency minimum as it seems to be only required for ruby 3.
weirdly enough it allows for a later version of jeweler by downgrading a few other gems.

Screenshot_20210407_063934

Comment (grantspeelman) [reply gem_config.rb:39]: maybe worth considering dropping jeweler for the bundler built in gem support

Comment (arunagw) [reply gem_config.rb:39]: I think `jeweler` can be dropped.

Comment (grantspeelman) [reply gem_config.rb:39]: should I drop it in this PR instead of setting minimum mime-types?

Comment (arunagw) [reply gem_config.rb:39]: Yes. I think it's better to drop unused gems.

**Comment by grantspeelman** [comment]
@arunagw good to merge?

**Comment by arunagw** [comment]
Thank you @grantspeelman for the work.

**Pull request merged**